.azure-pipelines/client.test.live.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   - job: 'Windows'
 
     pool:
-      vmImage: 'vs2017-win2016'
+      vmImage: 'windows-2019'
```

Comment:
windows-2022 has dropped python 3.6, since we want to support python 3.6 for now, let's use windows-2019